docs/en/query_language/alter.md

```diff
@@ -7,46 +7,118 @@ The `ALTER` query is only supported for `*MergeTree` tables, as well as `Merge`a
+Adding a column just changes the table structure, without performing any actions with data. The data doesn't appear on the disk after `ALTER`. If the data is missing for a column when reading from the table, it is filled in with default values (by performing the default expression if there is one, or using zeros or empty strings). The column appears on the disk after merging data parts (see [MergeTree](../operations/table_engines/mergetree.md)).
 
-This approach allows us to complete the ALTER query instantly, without increasing the volume of old data.
+This approach allows us to complete the `ALTER` query instantly, without increasing the volume of old data.
+
+Example:
 
 ``` sql
-DROP COLUMN name
+ALTER TABLE visits ADD COLUMN browser String AFTER user_id
 ```
 
-Deletes the column with the name 'name'.
+#### DROP COLUMN {#alter_drop-column}
+
+``` sql
+DROP COLUMN [IF EXISTS] name
+```
+
+Deletes the column with the name `name`. If the `IF EXISTS` clause is specified, the query won't return an error if the column doesn't exist.
 
 Deletes data from the file system. Since this deletes entire files, the query is completed almost instantly.
 
+Example:
+
 ``` sql
-CLEAR COLUMN name IN PARTITION partition_name
+ALTER TABLE visits DROP COLUMN browser
 ```
 
-Clears all data in a column in a specified partition.
+#### CLEAR COLUMN {#alter_clear-column}
 
 ``` sql
-MODIFY COLUMN name [type] [default_expr]
+CLEAR COLUMN [IF EXISTS] name IN PARTITION partition_name
 ```
 
-Changes the 'name' column's type to 'type' and/or the default expression to 'default_expr'. When changing the type, values are converted as if the 'toType' function were applied to them.
+Resets all data in a column for a specified partition. If the `IF EXISTS` clause is specified, the query won't return an error if the column doesn't exist.
+
+Example:
+
+``` sql
+ALTER TABLE visits CLEAR COLUMN browser IN PARTITION tuple()
+```
+
+#### COMMENT COLUMN {#alter_comment-column}
+
+``` sql
+COMMENT COLUMN [IF EXISTS] name 'comment'
+```
+
+Adds a comment to the column. If the `IF EXISTS` clause is specified, the query won't return an error if the column doesn't exist.
+
+Each column can have one comment. If a comment already exists for the column, a new comment overwrites the previous comment.
+
+Comments are stored in the `comment_expression` column returned by the [DESCRIBE TABLE](misc.md#misc-describe-table) query. 
+
+Example:
+
+``` sql
+ALTER TABLE visits COMMENT COLUMN browser 'The table shows the browser used for accessing the site.'
+```
+
+#### MODIFY COLUMN {#alter_modify-column}
+
+``` sql
+MODIFY COLUMN [IF EXISTS] name [type] [default_expr]
+```
+
+If the `IF EXISTS` clause is specified, the query won't return an error if the column doesn't exist.
```

Comment:
this "if exists" sentences should go after basic explanation of what commond does, not before